src/core/cs3iface.py

```diff
@@ -100,9 +104,11 @@ def statx(endpoint, fileid, userid, versioninv=0):
     return stat(endpoint, fileid, userid, versioninv)
 
 
-def setxattr(_endpoint, filepath, userid, key, value, lockid):
+def setxattr(endpoint, filepath, userid, key, value, lockid):
     '''Set the extended attribute <key> to <value> using the given userid as access token'''
-    reference = cs3spr.Reference(path=filepath)
+
+    # This assumes that the filepath is the same as the fileid like in the method above. Is that ok?
```

Comment:
Now this assumption is generally false: the filepath is always a path in all methods other than `stat()`.

I understand this is a step towards supporting spaces as endpoints, but we have to see how the WOPI core logic must be adapted. The current logic uses `stat()` at the very beginning to resolve the file reference to a `(endpoint, absoulte_path)` in all cases, can we keep this assumption?

Reply:
(in principle solved, see comment below)